### ISSUE TITLE
`/code/src_files/bin/docker-entrypoint.sh: line 14: True: command not found` when starting docker image

### ISSUE THREAD

**Issue by nam20485** [hackoregon/2019-backend-cookiecutter-django#8]
When running start.sh, the following error is produced:

```
$ bin/start.sh -d
Recreating 2019sandboxbackend_api_1 ... 
Recreating 2019sandboxbackend_api_1 ... done
Attaching to 2019sandboxbackend_api_1
api_1  | Debug: True
api_1  | /code/src_files/bin/docker-entrypoint.sh: line 14: True: command not found
api_1  | Postgres is up
```

The command on the following line is responsible for this output: 
https://github.com/hackoregon/2019-backend-docker/blob/da6543983623af5aa58f1c2bb079d54b4a6fbc45/bin/docker-entrypoint.sh#L14

The `if` statement does seem to function correctly, i.e it does not run the script on my development environment, so it may be a red herring.

**Comment by bhgrant8** [comment]
will take a look at what the correct pathing here is. 

I think it is falling back to the built in entrypoint in the docker. vs running the one in the cookiecutter template, which are generally the same, but not sure

**Comment by znmeb** [comment]
Yeah, I'm running into bizarre behavior with this script too - still troubleshooting. I'll add to this issue when I figure out what's going on.

**Comment by znmeb** [comment]
Previous comment was wrong - deleted. But there's a `command` in `docker-compose.yml` that may or not be necessary. I'll test it tonight when I get home. Meanwhile ...

https://medium.freecodecamp.org/docker-entrypoint-cmd-dockerfile-best-practices-abc591c30e21

**Comment by znmeb** [comment]
I think removing the `command` entry in the `docker-compose.yml` file gets rid of this.

**Comment by bhgrant8** [comment]
Should be fixed via #23, closing, please reopen if you see further issue
